CMakeLists.txt

```diff
@@ -1,39 +1,20 @@
 cmake_minimum_required(VERSION 3.0)
 
-option(WARN_VERBOSE "Enable highest warning level and disable warnings as errors" OFF)
+project(shaderunner)
 
+option(WARN_VERBOSE "Enable highest warning level and disable warnings as errors" OFF)
 
 set(SOURCE_DIR sources/src)
-set(INCLUDE_DIR sources/inc)
 set(LIB_DIR lib)
 
 # ==============================================================================
-
-set( LIB_INCLUDE_DIR
-	 ${LIB_DIR}/glew/include/
-	 ${LIB_DIR}/boost/
-	 ${LIB_DIR}/imgui/
-	 )
```

Comment:
This is now managed through `target_link_libraries` now that linked target declared properly their dependencies